test/tests/kernels/array_coupled_time_derivative/tests

```diff
@@ -0,0 +1,29 @@
+[Tests]
+  [./jac_1d]
```

Comment:
We will need a `design` and `issues` for this. This is why the documentation target is failing. If you need any help on how to add those, just let me know!

Reply:
@lindsayad I think I've gotten the `issues` and `design` sorted out. I basically copied `CoupledTimeDerivative` word for word with some small changes. Let me know if that does what you need.